e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -65,6 +66,10 @@ public void When_ExtractRequestReceived_Expect_ExtractStatusAndDocumentDataAdded
 
         var ehrContinue = (Document) waitFor(() -> Mongo.findEhrExtractStatus(conversationId).get(EHR_CONTINUE));
         assertThatExtractContinueMessageWasSent(ehrContinue);
+
+        TimeUnit.SECONDS.sleep(10);
```

Comment:
Don't add sleeps to tests it makes the run very slowly. There is almost always a better way.

Reply:
For example await for something to happen:
https://github.com/nhsconnect/integration-adaptor-lab-results/blob/07c8f5f247d41ab2c8e0ea872f6f222e24c16c7c/src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundUserAcceptanceTest.java#L132

Reply:
Thanks Implemented this